components/common/TableHeaders.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
 import Tooltip from './Tooltip'
+const { values } = require('ramda')
 const tableHeaders = require('../../lib/i18n/table-headers.json')
 const glossary = require('../../lib/i18n/glossary_en.json')
 
 const TableHeaders = ({ tableName }) => {
+  return values(prepareAllHeaders(tableName))
```

Comment:
Shall we revert changes in this file since they aren't being used with your recent commits?

Reply:
I think if we go ahead and do all the tables using the Table component, then we don't need the TableHeaders component or table-headers.json because they would be handled a) by the schema in each component b) by the Table component.

Reply:
I'm with you there